commands/team.go

```diff
@@ -354,16 +355,19 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 func restoreTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	teams := getTeamsFromTeamArgs(c, args)
+	var result *multierror.Error
 	for i, team := range teams {
 		if team == nil {
+			result = multierror.Append(result, errors.New("Unable to find team '"+args[i]+"'"))
```

Comment:
We can use formatted errors for a better readability.
```suggestion
			result = multierror.Append(result, fmt.Errorf("unable to find team %s", args[i]))
```

Reply:
Thanks. I have accepted the suggestions.